src/Scrolling.md

```diff
@@ -4,11 +4,19 @@
 These registers can be accessed even during Mode 3, but modifications may not take
 effect immediately (see further below).
 
-## FF42–FF43 — SCY, SCX: Viewport Y position, X position
+## FF42–FF43 — SCY, SCX: Background viewport Y position, X position
 
-Those specify the top-left coordinates of the visible 160×144 pixel area within the
+Specify the top-left coordinates of the visible 160×144 pixel area within the
```

Comment:
Maybe "these two registers", or "this pair of registers"? I don't think imperative works very well here.

Reply:
I think "These two registers" sounds better, yeah